sites/skeleton.dev/src/routes/(inner)/utilities/popups/+page.svelte

```diff
@@ -578,7 +578,7 @@ const popupState: PopupSettings = {
 		<section class="space-y-4">
 			<h2 class="h2">Combobox</h2>
 			<p>
-				The goal of Skeleton is to combine primitive elements and components to build more complex UI. For example, by combining a Button,
+				The goal of Skeleton is to combine primitive elements and components to build a more complex UI. For example, by combining a Button,
```

Comment:
I'd make the argument that *to build more complex user interfaces/UIs* sounds better here. It just sounds weird saying you're making "a UI", at least to me. Not a big deal tho.

Reply:
My correction was solely following the English grammar. It makes sense to me as in "a cross-platform UI".

Reply:
Don't stress about it guys. I'll review and make adjustments as needed. Thanks for the help and input to both of you though!